parsers/PA.py

```diff
@@ -174,9 +180,92 @@ def fetch_production(zone_key='PA', session=None, target_datetime=None, logger:
 
     return data
 
+def fetch_exchange(zone_key1='CR', zone_key2='PA', session=None,
+                   target_datetime=None, logger=logging.getLogger(__name__)) -> dict:
+    """
+    Requests the last known power exchange (in MW) between two countries.
+    """
+
+    if target_datetime:
+        raise NotImplementedError(
+            'This parser is not yet able to parse past dates')
```

Comment:
Suggest to match format in fetch_consumption:

```suggestion
    if target_datetime:
        raise NotImplementedError("This parser is not yet able to parse past dates")
```

This is longer than 80 characters, but many other lines here are also much longer than 80

Reply:
It now matches format in fetch_consumption as of commit 2b9097e.